test/integration/action/action-integration.spec.js

```diff
@@ -131,7 +131,7 @@ describe('Action Integration Tests', function() {
     });
 
     nav1 = sinon.createStubInstance(NavAction);
-    notify1 = sinon.createStubInstance(NotificationAction);
+    notify1 = sinon.createStubInstance(NotificationAction, nav1);
```

Comment:
Wondering if instead of labeling the two sets of everything as 1 and 2, a more descriptive variable name (or a comment)? I'm a little unclear on what each set is being used for

Reply:
We just have two lnd nodes and respective action instances for each node to keep state. Kinda like two lightning apps connecting two each other just without the react ui.